### ISSUE TITLE
Post-hooks in SQL no longer run in v0.11

### ISSUE THREAD

**Issue by tomdean** [fishtown-analytics/dbt#989]
### Issue description

Post-hooks in SQL files passed to the `config` Jinja2 macro no longer run.

```
{{
  config({
    "materialized": 'incremental',
    "sql_where": True,
    "unique_key": 'order_key',
    "post-hook": [
      'CREATE UNIQUE INDEX IF NOT EXISTS IX_{{this.name}}_pk ON {{ this }}(order_key)',
      'CREATE INDEX IF NOT EXISTS IX_{{this.name}}_placed_at ON {{ this }}(created_at)'
    ]
  })
}}
```

### Results

Expect two indexes to be created on the table, but none exists after either an incremental or full refresh. All indices across all tables & post-hooks are created as expected in 0.10.x

### System information
The output of `dbt --version`:
```
dbt --version
installed version: 0.11.0
   latest version: 0.11.0

Up to date!
```

The operating system you're running on:
```
uname -a
Darwin ****.local 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64
```

The python version you're using (probably the output of `python --version`)

```
python --version
Python 3.6.4
```

### Steps to reproduce

Create an index through through a post-hook passed to the config macro.

**Comment by drewbanin** [comment]
Thanks for the report @tomdean! This is a dupe of https://github.com/fishtown-analytics/dbt/issues/984 and was fixed in https://github.com/fishtown-analytics/dbt/pull/985.

It's currently in `development` and will be released in an 0.11.1 bugfix patch this coming week! Sorry for any inconvenience this may have caused.

**Comment by tomdean** [comment]
Missed those while searching around - thanks for the quick reply!
